libsolidity/interface/AssemblyStack.cpp

```diff
@@ -86,7 +87,8 @@ eth::LinkerObject AssemblyStack::assemble(Machine _machine) const
 	{
 	case Machine::EVM:
 	{
-		auto assembly = assembly::CodeGenerator::assemble(*m_parserResult, *m_analysisInfo);
+		eth::Assembly assembly;
```

Comment:
This is already included in this file, so in my opinion it is cleaner using it directly.